cmake/Modules/FindRSSDK2.cmake

```diff
@@ -0,0 +1,16 @@
+###############################################################################
+# Find Intel RealSense SDK 2.0 (librealsense)
+#
+#     find_package(RSSDK2)
+#
+# Variables defined by this module:
+#
+#  RSSDK2_FOUND                 True if RealSense SDK 2.0 was found
+#  RSSDK2_INCLUDE_DIRS          The location(s) of RealSense SDK 2.0 headers
+#  RSSDK2_LIBRARIES             Libraries needed to use RealSense SDK 2.0
+
+find_package(realsense2)
+
+set(RSSDK2_FOUND ${realsense2_FOUND})
+set(RSSDK2_INCLUDE_DIRS ${realsense2_INCLUDE_DIR})
+set(RSSDK2_LIBRARIES ${realsense2_LIBRARY})
```

Comment:
Many <code>cmake/Modules/Find***.cmake</code> seem to report that library has been found to user using output status in last.
https://github.com/PointCloudLibrary/pcl/blob/52eb8a154433aa21e19acecdbba055ceaf4d5961/cmake/Modules/FindEigen.cmake#L39-L41
```
Eigen found (include: C:/Program Files/Eigen3/include/eigen3, version: 3.3.7)
```
We need consider to add output status. What do you think?
```cmake
if(RSSDK2_FOUND)
  message(STATUS "RealSense SDK 2 found (include: ${RSSDK2_INCLUDE_DIRS}, lib: ${RSSDK2_LIBRARIES}, version: ${realsense2_VERSION})")
endif()
```
```
RealSense2 found (include: C:/Program Files/librealsense2/include, lib: realsense2::realsense2, version: 2.23.0)
```